build/localnet/dependencies/values.yaml

```diff
@@ -64,9 +59,3 @@ prometheus:
   url: http://{{ include "prometheus.fullname" .}}:{{ .Values.prometheus.server.service.servicePort }}{{ .Values.prometheus.server.prefixURL }}
   datasource:
     jsonData: {}
-
-# https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml
-postgres:
```

Comment:
Why can we remove this?

Reply:
@Olshansk because we no longer use one large Postgres instance but provision new, ephemeral Postgres for each validator individually. That way, both – storage volume and Postgres database – gets erased on scale down.